pkgs/jovian-greeter/greeter.py

```diff
@@ -87,6 +87,9 @@ def create_session(self, username: str):
     def start_session(self, command: List[str], environment: List[str]):
         try:
             subprocess.check_call(["plymouth", "quit", "--retain-splash", "--wait"])
+        except FileNotFoundError:
+            logging.debug("Failed to stop Plymouth: could not find executable")
```

Comment:
That's not going to work, the greeter is always wrapped with the Plymouth executable.

Reply:
You told me on Matrix that the error is benign. Did I misunderstand something?

Reply:
The error is indeed benign, but `FileNotFoundError` will never actually trigger here.